RadarSDK/Radar.m

```diff
@@ -161,6 +161,7 @@ + (void)trackOnceWithDesiredAccuracy:(RadarTrackingOptionsDesiredAccuracy)desire
                                                      NSArray<RadarGeofence *> *_Nullable nearbyGeofences, RadarConfig *_Nullable config, NSString *_Nullable token) {
                                      if (status == RadarStatusSuccess) {
                                          [[RadarLocationManager sharedInstance] replaceSyncedGeofences:nearbyGeofences];
+                                         [[RadarLocationManager sharedInstance] updateTrackingFromMeta:config.meta];
```

Comment:
This `config` is nullable so we should check that exists before accessing `config.meta`